test/unit/widgets/forms.test.ts

```diff
@@ -1,1435 +1,1455 @@
-//   it('exists', () => {
-//     expect(newThing).toBeInstanceOf(Object)
-//   })
-//   it('runs', () => {
-//     expect(newThing(RdfLib.sym(''))).toBeInstanceOf(Object)
-//   })
-//   // need to also mock or figure out doc - which is a NamedNode
-//   it.skip('returns the correct .', () => {
-//     const Date = jest.fn()
-//     Date.mockReturnValueOnce('Thu Feb 06 2020 19:42:59 GMT+1100')
-//     expect(newThing('doc')).toBe('')
-//   })
-// })
+import { namedNode, graph } from 'rdflib'
+import ns from '../../../src/ns'
+import uiStore from '../../../src/store'
+
+import {
+  appendForm,
+  buildCheckboxForm,
+  editFormButton,
+  field,
+  fieldFunction,
+  fieldLabel,
+  fieldParams,
+  fieldStore,
+  findClosest,
+  formsFor,
+  makeDescription,
+  makeSelectForCategory,
+  makeSelectForNestedCategory,
+  makeSelectForOptions,
+  mostSpecificClassURI,
+  newButton,
+  newThing,
+  promptForNew,
+  propertiesForClass,
+  sortByLabel,
+  sortBySequence
+} from '../../../src/widgets/forms'
+
+// jest.mock('rdflib')
+// jest.mock('solid-auth-client')
+
+describe('field', () => {
+  it('exists', () => {
+    expect(field).toBeInstanceOf(Object)
+  })
+})
+
+describe('Form field', () => {
+  it('exists', () => {
+    expect(field[ns.ui('Form').uri]).toBeInstanceOf(Object)
+  })
+  it('runs', () => {
+    const container = document.createElement('div')
+    const already = {}
+    const subject = namedNode('http://example.com/#this')
+    const form = namedNode('http://example.com/#form')
+    const store = namedNode('http://example.com/#store')
+    const callbackFunction = () => {}
+
+    // FIXME: https://github.com/solid/solid-ui/issues/239
+    ;(document as any).outlineManager = {
+      appendPropertyTRs: () => {}
+    }
+    field[ns.ui('Form').uri](
+      document,
+      container,
+      already,
+      subject,
+      form,
+      store,
+      callbackFunction
+    )
+
+    expect(
+      field[ns.ui('Form').uri](
+        document,
+        container,
+        already,
+        subject,
+        form,
+        store,
+        callbackFunction
+      )
+    ).toMatchInlineSnapshot(`
+     <div
+       style="padding-left: 2em; border: 0.05em solid #888888;"
+     />
+    `)
+  })
+  // @@ TODO check this further what test could I use to make sure
+  // to test that the form gets added but obviously not in the container passed in
+  it.skip('still returns an element if the container is null', () => {
+    // debugger
+    const container = document.createElement('div')
+    const already = {}
+    const subject = namedNode('http://example.com/#this')
+    const form = namedNode('http://example.com/#form')
+    const store = namedNode('http://example.com/#store')
+    const callbackFunction = () => {}
+    expect(
+      field[ns.ui('Form').uri](
+        document,
+        container,
+        already,
+        subject,
+        form,
+        store,
+        callbackFunction
+      )
+    ).toMatchInlineSnapshot(`
```

Comment:
Please replace all instances toMatchInlineSnapshot with toMatchSnapshot.

(And also remove the use of Prettier - don't know how that got back in there.)